curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md

```diff
@@ -16,13 +16,16 @@ Change the font color of all the anchor elements within the list elements to som
 You should use the `li > a` selector.
 
 ```js
-assert.exists(new __helpers.CSSHelp(document).getStyle('li > a'));
+const anchors = document.querySelector('li > a'); 
+assert.exists(getComputedStyle(anchors));
 ```
 
 You should give the `a` element a `color` property.
 
 ```js
-assert.notEmpty(new __helpers.CSSHelp(document).getStyle('li > a')?.color);
+const anchors = document.querySelector('li > a'); 
```

Comment:
this gets only the first `li > a`, not all of them. Maybe you can use `querySelectorAll`, and check that the style exist for each `li > a`? Or is it unnecessary?

Reply:
I probably should check all of them.